test/onnx/test_utility_funs.py

```diff
@@ -1042,6 +1042,21 @@ def forward(self, x, y, z):
         for node in graph.nodes():
             self.assertIn(node.scopeName(), expected_scope_names)
 
+        expected_torch_script_scope_names = {
+            "test_utility_funs.M::/torch.nn.modules.activation.GELU::gelu1",
+            "test_utility_funs.M::/torch.nn.modules.activation.GELU::gelu2",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.0",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.1",
+            "test_utility_funs.M::/torch.nn.modules.normalization.LayerNorm::lns.2",
+            "test_utility_funs.M::",
+        }
+
+        graph, _, _ = self._model_to_graph(
+            torch.jit.script(model), (x, y, z), input_names=[], dynamic_axes={}
+        )
+        for node in graph.nodes():
+            self.assertIn(node.scopeName(), expected_torch_script_scope_names)
```

Comment:
nit (I guess): I can see the reason that a hardcoded expected outcomes are used here, just like we have under expect folder. I am concerned that what if there is a change in onnx graph that the nodes are not what we expect now, but the naming logic is still right. In that case, we will have to manually change this hardcoded part here again. But I might be wrong that there shouldn't be this major different in onnx graph?

Reply:
It's legitimate concern. Hence the test is written with simple nodes such that the nodes rarely change, and test failures are mostly triggered by name discrepancies.